script/Arcueid/Day 8/08_00_ARC08_3_1.txt

```diff
@@ -184,12 +184,12 @@ I didn't want to waste your time again like yester―――"
 -- Page 172, Offset 20096.
 -- 「それこそ余分な気遣いだっての。協力すると言った以上、どんな作業だって一緒にやるんだから夜まで待っててくれよ。
 "I'm telling you there's no need to go that far.
- I said I'd team up with you, so whatever needs doing just hold on until tonight then let's do it together, okay?
+ I said I'd team up with you, so whatever needs doing just hold on until tonight, then let's do it together, okay?
```

Comment:
I said I'd team up with you so just hold on until tonight. We can settle whatever needs to be done together, okay?